ci/dev/postinstall.sh

```diff
@@ -3,17 +3,19 @@ set -euo pipefail
 
 main() {
   cd "$(dirname "$0")/../.."
-  source ./ci/lib.sh
 
-  # This installs the dependencies needed for testing
+  echo 'Installing code-server test dependencies...'
+
   cd test
-  yarn
+  yarn install
   cd ..
 
-  cd lib/vscode
-  yarn ${CI+--frozen-lockfile}
+  cd vendor
+  echo 'Installing vendor dependencies...'
 
-  symlink_asar
+  # Ignore scripts to prevent partial install which omits development dependencies.
+  yarn install --modules-folder modules --ignore-scripts
```

Comment:
Ah, this is probably where we are missing that `--frozen-lockfile`.

I am not clear on why we need to ignore scripts; how do they cause a partial install?